apps/patient/src/views/Pharmacy.tsx

```diff
@@ -55,9 +55,9 @@ export const Pharmacy = () => {
   const [searchParams] = useSearchParams();
   const token = searchParams.get('token');
   const isReroute = searchParams.get('reroute');
+  const pharmacyClosed = searchParams.get('pharmacyClosed');
```

Comment:
what if we flip it and make it `openNow`? that way it's applicable to the filters and later we could do something similar with `open24Hours` if we want to navigate directly to this view and auto-enable filters

Reply:
I like that